### PR TITLE
updated daos, copper, data mover and filesystem docs

### PR DESCRIPTION
**Pull request by kaushikvelusamy**
updated daos, copper, data mover and filesystem docs

### CODE REVIEW

File: docs/aurora/data-management/copper/copper.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+* Copper currently does not support write operation. 
+* Only the follow file system operations are supported : init, open, read, readdir, readlink, getattr, ioctl, destroy
+* Copper works only from the compute nodes and you need a minimum of 2 nodes up to a max of any number of nodes ( Aurora max 10624 nodes)
+* Recommended trees is 1 or 2. 
+* Recommended size for max cachable byte size is 10MB to 100MB.
+* To be used only from the compute node.
+* More examples at https://github.com/argonne-lcf/copper/tree/main/examples/example3 
+
+
```
Comment (zhenghh04) [inline docs/aurora/data-management/copper/copper.md:81]: @kaushikvelusamy please add link to the readthedoc documentation.

File: docs/aurora/data-management/daos/daos-overview.md [hunk 2]
```diff
@@ -45,154 +66,303 @@ Total size: 6.0 TB
...
-exit 0
 ```
 
-## Job Submission
 
-The above job script expects two environment variables which you set to the relevant pool and container.
-The `-ldaos=default` switch will ensure that DAOS is available on the compute node.
+## Sample job script
+
+Currently, ``--no-vni`` is required in the ``mpiexec`` command to use DAOS.
```
Comment (zhenghh04) [inline docs/aurora/data-management/daos/daos-overview.md:225]: I added this statement about --no-vni. Please check whether "--no-vni" is still required.

File: docs/aurora/data-management/daos/daos-overview.md [hunk 3]
```diff
@@ -45,154 +66,303 @@ Total size: 6.0 TB
...
+CPU_BINDING1=list:4:9:14:19:20:25:56:61:66:71:74:79
+```
+![Sample NIC to Core binding](core-NIC-binding.png "Sample NIC to Core binding")
+
+
+
+## Interception library for posix containers
+
+The interception library (IL) is a next step in improving DAOS performance. This provides kernel-bypass for I/O data, leading to improved performance.
+The IL will intercept basic read and write POSIX calls while all metadata calls still go through dFuse.
```
Comment (zhenghh04) [inline docs/aurora/data-management/daos/daos-overview.md:202]: For libpil4dfs.so, meta data will also be intercepted?

File: docs/aurora/data-management/daos/daos-overview.md [hunk 4]
```diff
@@ -202,77 +372,87 @@ Each DAOS server nodes is based on the Intel Coyote Pass platform.
...

+export DYLD_FALLBACK_LIBRARY_PATH=/Users/kvelusamy/Desktop/tools/spack/opt/spack/darwin-ventura-m1/apple-clang-14.0.3/darshan-util-3.4.4-od752jyfljrrey3d4gjeypdcppho42k2/lib/:$DYLD_FALLBACK_LIBRARY_PATH
+
+darshan-parser ~/Downloads/kaushikv_ior_id917110-44437_10-23-55830-632270104473632905_1.darshan 
+python3 -m darshan summary ~/Downloads/kaushikv_ior_id917110-44437_10-23-55830-632270104473632905_1.darshan #coming soon
+
+```
+
+## Cluster Size
+
```
Comment (zhenghh04) [inline docs/aurora/data-management/daos/daos-overview.md:434]: Should this be "Number of available DAOS servers"

File: docs/aurora/running-jobs-aurora.md [hunk 5]
```diff
@@ -10,6 +10,9 @@ There is a single routing queue in place called `EarlyAppAccess` which submits t
 - `lustre_scaling` (execution queue) : 10 running jobs per-user; max walltime : 6 hours; max nodecount : 9090 (subject to change)
 
 ### Submitting a job
+
+Note: Jobs should be submitted only from your allocated project directory and not from your home directory. 
+
```
Comment (zhenghh04) [inline docs/aurora/running-jobs-aurora.md:15]: Not so clear about what you want to say here. 

It is perfectly fine to submit jobs from the home directory. It just the data is suggested to be in Lustre or DAOS, right?

**Review by zhenghh04 — COMMENTED** [review]
@kaushikvelusamy please check the comments.

File: docs/aurora/running-jobs-aurora.md (hunk 5 above)
Comment (kaushikvelusamy) [reply docs/aurora/running-jobs-aurora.md:15]: removed.

File: docs/aurora/data-management/daos/daos-overview.md (hunk 4 above)
Comment (kaushikvelusamy) [reply docs/aurora/data-management/daos/daos-overview.md:434]: updated.

File: docs/aurora/data-management/daos/daos-overview.md (hunk 3 above)
Comment (kaushikvelusamy) [reply docs/aurora/data-management/daos/daos-overview.md]: updated.

File: docs/aurora/data-management/copper/copper.md (hunk 1 above)
Comment (kaushikvelusamy) [reply docs/aurora/data-management/copper/copper.md:85]: updated.

File: docs/aurora/data-management/copper/copper.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+If you want your I/O to go through copper, add /tmp/${USER}/copper/ to the begining of your PATHS. Here only the root compute node will do the I/O directly with the lustre file system. 
+If /tmp/${USER}/copper/ is not added to the begining of your paths, then all compute nodes would do I/O directly to the lustre file system. 
+
+Python Example
+
+```bash
+time mpirun --np ${NRANKS} --ppn ${RANKS_PER_NODE} --cpu-bind=list:4:9:14:19:20:25:56:61:66:71:74:79 --genvall \
+            --genv=PYTHONPATH=/tmp/${USER}/copper/lus/flare/projects/Aurora_deployment/kaushik/copper/oct24/copper/run/copper_conda_env \
+            python3 -c "import numpy; print(numpy.__file__)"
```
[the remark this answers is not in the document]
Comment (kaushikvelusamy) [reply docs/aurora/data-management/copper/copper.md]: updated.

**Review by zhenghh04 — APPROVED** [review]

**Pull request merged**